roles/scp_http_int/tasks/user_creation.yml

```diff
@@ -0,0 +1,6 @@
+---
+- user: name=logs state=present
+  name: Creates a user
+
+- file: path=/home/logs/glusterfs-logs state=directory
```

Comment:
Permission would likely be wrong on that folder, since that's running as root, and we want the file to be writable by the unprivileged user.

Reply:
And in fact, it would IMHO be better to put the folder in /var/www, since that wouldn't requires to touch to selinux policy.

We also need to make sure that is some form of quota. Either by using a separate partition, or using regular quota.